### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by ps-jarvis**

### CODE REVIEW

**Review by Hlavtox — CHANGES_REQUESTED** [review]
Something is wrong, all of those were translated.

**Comment by Quetzacoalt91** [comment]
This is caused by the renaming of templates, the context does not match perfectly and the translations were not transferred. I'll have a look on the UI.

File: translations/ModulesAutoupgradeAdmin.fr.xlf [hunk 1]
```diff
@@ -2116,17 +2121,17 @@ Comment: check DB access</note>
...
       <trans-unit id="b77d20ed1ba7b2ee7139dc4f3fd13117" approved="yes">
         <source>Allow major upgrade:</source>
-        <target state="final">Autoriser les mises à jour majeures :</target>
+        <target state="final">Autoriser les mise à jour majeure :</target>
         <note>Line: 101</note>
       </trans-unit>
       <trans-unit id="780daf2510c5c9e1c58948f138629817" approved="yes">
         <source>Archive to use:</source>
-        <target state="final">Archive à utiliser :</target>
+        <target state="final">Archive a utiliser :</target>
```
Comment (Quetzacoalt91) [inline translations/ModulesAutoupgradeAdmin.fr.xlf:2134]: There is an issue where old translations have been taken.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
The issue has been fixed on Crowdin. I triggered the TM feature to reapply the translations that existed before the update.

I also updated the project config in an attempt to have them automatically applied:
* Changed the TM priorities by having PrestaShop-official-module with an higher priority than PrestaShop-official
* Limited the conditions to apply an existing translation. We were expecting Perfect matches, from now we want 100% match.

**Review by Hlavtox — APPROVED** [review]
Looks good now @Quetzacoalt91, good job! 🫡

**Pull request merged**